package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-footer",
   "description": "Fozzie footer – Footer Component for Just Eat projects",
-  "version": "1.8.2",
+  "version": "1.8.3",
```

Comment:
Should bump the minor version here, rather than the patch, to become v1.9.0.

Reply:
@DamianMullins updated

Reply:
Hi @annieguo the version needs to be updated in both places, sorry!

Reply:
@DamianMullins 
Sorry, my bad, I meant to change both files. 
Now both updated.
Cheers.